wifite/model/target.py

```diff
@@ -134,9 +134,9 @@ def to_str(self, show_bssid=False):
         power = Color.s('{%s}%s' % (color, power))
 
         wps = Color.s('{O} n/a')
-        if self.wps == True:
+        if self.wps:
             wps = Color.s('{G} yes')
-        elif self.wps == False:
+        elif not self.wps:
             wps = Color.s('{O}  no')
         elif self.wps is None:
```

Comment:
This if-statement would never run, because `not None` evaluates to `True`.

Reply:
Hmmmm, i have just made a quick test to resolve the situation and find possible solution - please try Yourself replacing the WPS value with "True/False/None": 

`WPS = None`
`if WPS:`
`    print(True)`
`elif not WPS and WPS is not None:`
`    print(False)`
`elif WPS is None:`
`    print(None)`